### ISSUE TITLE
Fontawesome Pro

### ISSUE THREAD

**Issue by nicolez9911** [DirkPersky/typo3-rte-ckeditor-fontawesome#8]
Is there a possibility to use the plugin with FA pro Icons?

**Comment by DirkPersky** [comment]
Not yet, but if I have time again, I wanted to update the icons and integrate the Pro icons as an alternative option.

**Comment by Chriz74** [comment]
+1

**Comment by DirkPersky** [comment]
If you can test it, with the latest Dev version.
That would be great. Because I  have't a pro licence

## Use Fontawesome Pro:
first create the file `/fileadmin/rte_ckeditor_fontawesome/FA5Pro.yaml`.
Copy the content from below and replace your `fontawesome pro` link.
```
imports:
  - { resource: "EXT:rte_ckeditor/Configuration/RTE/Full.yaml" }
  - { resource: "EXT:rte_ckeditor_fontawesome/Configuration/RTE/PluginFA5.yaml" }
# Add configuration for the editor
editor:
  config:
    # css definitions for the editor
    contentsCss:
      - "EXT:rte_ckeditor/Resources/Public/Css/contents.css"
      - "LINK TO FONTAWESOME PRO"
```

Add to your TSConfig:
```
// Zuweisen der Konfiguration im PageTSconfig
RTE.default.preset = fa5pro
```

Add to your constant or clear this:
```
# Include Fontawesome 5-Pro CSS
plugin.tx_ckeditor_fontawesome.css = "LINK TO FONTAWESOME PRO"
```

**Comment by jpmschuler** [comment]
Adding this with version 2111221 creates an error after clicking on the RTE icon for your plugin:
<img width="410" alt="Bildschirmfoto 2019-06-18 um 11 26 09" src="https://user-images.githubusercontent.com/12411176/59670330-109b4080-91bc-11e9-8b69-d1873d56f2a2.png">
Am I missing something?

**Comment by DirkPersky** [comment]
Which config do you use?
The normal FA5 without Pro or the Pro configuration?

I've installed the latest dev-master version for my website and it seems to work

**Comment by jpmschuler** [comment]
At first I tried to incorporate it in my current configuration, which resulted in the error. But removing my config and using yours results in the very same behavior:
Page.TsConfig
```
RTE.default.preset = fa5pro
```
TypoScript 
```
plugin.tx_ckeditor_fontawesome.css = https://pro.fontawesome.com/releases/v5.9.0/css/all.css
```
```
imports:
  - { resource: "EXT:rte_ckeditor/Configuration/RTE/Full.yaml" }
  - { resource: "EXT:rte_ckeditor_fontawesome/Configuration/RTE/PluginFA5Pro.yaml" }
# Add configuration for the editor
editor:
  config:
    # css definitions for the editor
    contentsCss:
      - "EXT:rte_ckeditor/Resources/Public/Css/contents.css"
      - "https://pro.fontawesome.com/releases/v5.9.0/css/all.css"
```

Neither external like above nor internal CSS like `EXT:mysiteext/Resources/Public/Stylesheets/Additional/fontawesome-icons.min.css`works here in 8LTS, both result in the same error. I'd be happy to debug, but I just get the screenshot above, so the  broken 
```html
<td class="cke_dialog_ui_vbox_child" role="presentation"> id="cke_95_uiElement" class="cke_dialog_ui_html" </td>
```

**Comment by DirkPersky** [comment]
It also happens if you use the normal non-FA5 Pro configurations:

`` `
RTE.default.preset = full
`` `

Or can you give me access to your system so that I can possibly debug it?

dirk.persky@gmail.com

**Comment by jpmschuler** [comment]
The above mentioned error comes up only in OSX Safari:
```
imports:
  - { resource: "EXT:rte_ckeditor_fontawesome/Configuration/RTE/PluginFA5Pro.yaml" } 
```
 If I change the import from that to
```
imports:
  - { resource: "EXT:rte_ckeditor_fontawesome/Configuration/RTE/PluginFA5.yaml" }
```
everything works as always. The popup shows the icon selector as normal. As soon as I use the PRO the above error occurs as popup content.

In OSX Chrome it seems to work.

**Comment by jpmschuler** [comment]
Additionally: The Pro CSS seems to use an old FA version, thus some icons don't show:
Resources/Public/JavaScript/Plugins/ckeditor_fa5pro/dialogs/index.html
```html
    <link href="https://use.fontawesome.com/releases/v5.0.6/css/all.css" rel="stylesheet">
```

**Comment by DirkPersky** [comment]
Fixed that isue

**Comment by jpmschuler** [comment]
As you closed this issue, I tried v9.5.2, works like a charm in Chrome - with SafariOSX still the issue above. Could you confirm that it should work in SafariOSX and this must be a local issue or should I reopen a more specific issue.
